packages/cosmwasm-stargate/src/cosmwasmclient.ts

```diff
@@ -264,7 +264,9 @@ export class CosmWasmClient {
     const pollForTx = async (txId: string): Promise<DeliverTxResponse> => {
       if (timedOut) {
         throw new TimeoutError(
-          `Transaction with ID ${txId} was submitted but was not yet found on the chain. You might want to check later.`,
+          `Transaction with ID ${txId} was submitted but was not yet found on the chain. You might want to check later. There was a wait of ${
+            timeoutMs / 10000
+          } seconds. Approx. ${timeoutMs / pollIntervalMs} blocks were created during this time.`,
```

Comment:
We don't have any information on the block time unfortunately. The `pollIntervalMs` is only the time after which the client polls again to check for the block.

The two information we do have is: time out, time after which we gave up. This can be e.g. 10 seconds timeout and 12 seconds for the runtime. The time after we gave up is not available directly now. We have to store a start time in line 260.

Reply:
So isn't the first sentence enough? `There was a wait of ${
            timeoutMs / 10000
          } seconds.`

Reply:
Yeah, that's enough and simple